ceph-releases/jewel/ubuntu/14.04/daemon/entrypoint.sh

```diff
@@ -556,20 +562,26 @@ function osd_activate {
     timeout 10  bash -c "while [ ! -e ${OSD_DEVICE} ]; do sleep 1; done"
     chown ceph. ${OSD_JOURNAL}
     if [[ ${OSD_DMCRYPT} -eq 1 ]]; then
-      ceph-disk -v --setuser ceph --setgroup --dmcrypt disk activate $(dev_part ${OSD_DEVICE} 1)
+      ceph-disk -v --setuser ceph --setgroup --dmcrypt disk activate $(dev_part ${OSD_DEVICE} 1) ${CEPH_DISK_ACTIVATE_OPTS}
```

Comment:
shouldn't that be "--setgroup disk  --dmcrypt"? Might as well fix it while your here